WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -39,9 +39,9 @@ def create
       end
       redirect_to edit_competition_path(@competition)
     else
-      # Show id errors under name, since we don't actually show an
+      # Show friendly id errors under name, since we don't actually show an
       # id field to the user, so they wouldn't see any id errors.
```

Comment:
nitpick: `id field` is mentioned twice here, maybe it should also be changed to `friendly id` or `friendly_id`?